trip-ninja-inc/trip_ninja_api_docs
# Feature/PROD-1292 | Update api docs to reflect new msdp search functionality

## LocalQA approved by
@Sanpele 

## Description
- Updated the MSDP search documentation including
  - Search
  - Get Flight results
  - Get Hotel results
  - Remove hotel results 

## How to Test
1. Checkout this branch
2. Run  `npx @redocly/openapi-cli preview-docs tn_api_msdp_v1.yml`
3. Go to http://127.0.0.1:8080
4. You should see the documentation for the mentioned endpoints above in `Description`
